### PR TITLE
Unmatch CTA border color needs to be fixed in new mobile app

### PR DESCRIPTION
**Pull request by hlkavya0213**
Screenshot 2021-04-05 at 12 21 59 PM

### CODE REVIEW

File: src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss [hunk 1]
```diff
@@ -158,13 +158,13 @@
...
+    text-transform: capitalize;
     font-weight: 700;
-    border-radius: 8px;
-    margin: 14px;
+    margin: 14px !important;
     width: 100%;
     margin: 12px 8px;
-    --border-radius: 4px !important;
+    border-radius: 4px;
+    border: 1px solid #CCCDD7;
```
Comment (Julias0) [inline src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss:167]: lowercase please

**Review by Julias0 — APPROVED** [review]

File: src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss (hunk 1 above)
Comment (iamtarun1993) [reply src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss:167]: declare this as variable first and then use it.

File: src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss [hunk 2]
```diff
@@ -158,13 +158,13 @@
   &--primary-cta {
     height: 48px;
     font-size: 14px;
-    text-transform: uppercase;
+    text-transform: capitalize;
```
Comment (iamtarun1993) [inline src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss:161]: add `text-capitalize` class in html

Comment (iamtarun1993) [reply src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss:161]: remove it from here

File: src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.ts [hunk 3]
```diff
@@ -51,7 +51,7 @@ export class CccClassifiedActionsPage implements OnInit {
            return !!cccExpense.matched_by &&
              cccExpense.state !== 'SETTLED' &&
              cccExpense.state === 'IN_PROGRESS' &&
-             matchedExpense.every((expense) => ['COMPLETE', 'DRAFT', 'APPROVER_PENDING'].indexOf(expense.state) > -1);
```
Comment (hlkavya0213) [inline src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.ts:54]: This is coz we should be allowing to unmatch only from DRAFT & FYLED state

**Review by iamtarun1993 — APPROVED** [review]

File: src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss [hunk 4]
```diff
@@ -158,13 +160,12 @@
   &--primary-cta {
     height: 48px;
     font-size: 14px;
-    text-transform: uppercase;
     font-weight: 700;
-    border-radius: 8px;
-    margin: 14px;
+    margin: 14px !important;
```
Comment (abhishek1234321) [inline src/app/fyle/ccc-classified-actions/ccc-classified-actions.page.scss:164]: can we avoid `!Important`?

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**